### PR TITLE
items :: FileList -> Array File

### PR DESCRIPTION
**Pull request by BebeSparkelSparkel**
easier to get files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ package-lock.json
 /bower_components/
 /node_modules/
 /output/
+*.swp
```
Comment (hdgarrood) [inline .gitignore]: These entries should really go into your global `.gitignore` file, not into repo-specific ones.

**Comment by BebeSparkelSparkel** [comment]
#7

**Comment by garyb** [comment]
You've basically implemented a `toUnfoldable` here, so maybe we should do that instead? Then consumers can choose whether they want an array, list, or whatever. 

Despite what I just said in #7, I would be fine with adding a utility like this, I just don't want to replace the whole API with an Array or List representation.

**Comment by BebeSparkelSparkel** [comment]
@garyb thanks for `toUnfoldable`. I have found pursuit type search to be inferior to hoogle and have had trouble finding functions.

**Comment by BebeSparkelSparkel** [comment]
@garyb I have switched to using `unfoldr`

File: src/Web/File/FileList.purs [hunk 2]
```diff
@@ -20,3 +23,7 @@ foreign import _item :: Int -> FileList -> Nullable File
 -- | Get `File` at the certain position
 item :: Int -> FileList -> Maybe File
 item i = toMaybe <<< _item i
+
+items :: FileList -> Array File
```
Comment (garyb) [inline src/Web/File/FileList.purs]: This could also be `forall t. Unfoldable t => FileList -> t File` now.

**Comment by BebeSparkelSparkel** [comment]
should be good now

**Review by the-dr-lazy — APPROVED** [review]

**Comment by garyb** [comment]
Thanks!

**Pull request merged**